config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
@nenioscio Can you revise your pull request to do the changes we agreed to?  Otherwise, I can do it.  Let me know.  Thanks!  

Note, we can worry about automatically restarting memcached in a followup.  I don't think it's hard but we'd need to ensure our session based workers either reconnect or we need to have them restart too.  It's not a single line change.